src/main/kotlin/no/nav/medlemskap/inst/lytter/service/JoarkService.kt

```diff
@@ -21,9 +23,10 @@ class JoarkService(
     }
     suspend fun handle(record : MedlemskapVurdertRecord)
     {
-        if (validateRecord(record)){
+        val medlemskapVurdering = JaksonParser().parseToObject(record.json)
+        if (validateRecord(medlemskapVurdering) && medlemskapVurdering.datagrunnlag.ytelse == Ytelse.LOVME_GCP.name){
```

Comment:
Bør vi validere på en liste over ytelser vi ønsker skal opprette PDF? Tenker fremover på AAP og Barnepensjon.  De vil ha identisk problemstilling der vi kommer til å ønske å kontrollere pdf`er som opprettes for dem også